src/main/java/io/stargate/sgv2/jsonapi/api/configuration/ObjectMapperConfiguration.java

```diff
@@ -58,15 +60,20 @@ private ObjectMapper createMapper(DocumentLimitsConfig documentLimitsConfig) {
              */
             .disable(StreamWriteFeature.WRITE_BIGDECIMAL_AS_PLAIN)
             .build();
+    // For now, just one custom serializer to register
+    SimpleModule cqlSerializersModule =
+        new SimpleModule("CqlSerializersModule").addSerializer(new CqlVectorSerializer());
```

Comment:
nit: Should this be called CqlVectorSerializersModule?

Reply:
Hmmh. For current state, yes, `CqlVectorSerializerModule`. Can change name if more serializers (or other things) need registered.